src/main/java/com/spotinst/sdkjava/example/ElastigroupUsageExampleAzure.java

```diff
@@ -30,15 +31,15 @@ public static void main(String[] args) throws IOException {
 
         // Sleep for provisioning
         System.out.println("Sleeping... waiting for provisioning 7 seconds.");
```

Comment:
Replace 7 seconds to 60 seconds in message

Reply:
changed